package/libstorage-ng.changes

```diff
@@ -1,17 +1,4275 @@
--------------------------------------------------------------------
```

Comment:
I see there is some old entries in changes. If there is agreement on it, then I am fine with it.